internal/refactoring/move_validate.go

```diff
@@ -234,7 +244,21 @@ func moveableObjectExists(addr addrs.AbsMoveable, in instances.Set) bool {
 		return in.HasResource(addr)
 	default:
 		// The above cases should cover all of the AbsMoveable types
-		panic("unsupported AbsMovable address type")
+		panic("unsupported AbsMoveable address type")
+	}
+}
+
+func resourceTypesDiffer(absFrom, absTo addrs.AbsMoveable) bool {
+	switch absFrom.(type) {
+	case addrs.AbsResourceInstance, addrs.AbsResource:
```

Comment:
I think you could say just `case addrs.AbsMoveableResource:` here and get the same effect, right?